server/src/main/java/org/apache/druid/segment/handoff/CoordinatorBasedSegmentHandoffNotifier.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.apache.druid.segment.realtime.plumber;
+package org.apache.druid.segment.handoff;
```

Comment:
I think we should change the log level of some info logs to debug in this class which are printed per segment. It could be a lot in batch ingestion.